test/built-ins/Temporal/PlainDateTime/prototype/subtract/hour-overflow.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.subtract
+description: Testing overflow hours (subtracting hours that push one to the next/previous day)
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+const later = new Temporal.PlainDateTime(2019, 10, 29, 10, 46, 38, 271, 986, 102);
+
+TemporalHelpers.assertPlainDateTime(
+  later.subtract({ hours: 12 }),
+  2019, 10, "M10", 28, 22, 46, 38, 271, 986, 102,
+  "subtract result"
+);
+
+const earlier = new Temporal.PlainDateTime(2019, 10, 29, 10, 46, 38, 271, 986, 102);
+
+TemporalHelpers.assertPlainDateTime(
+  earlier.add({ hours: -12 }),
+  2019, 10, "M10", 28, 22, 46, 38, 271, 986, 102,
+  "symmetrical with regard to negative durations (1)"
```

Comment:
Same comment as above about the assertion message

Reply:
fixed